src/librustc_error_codes/error_codes/E0741.md

```diff
@@ -9,12 +10,15 @@ struct A;
 struct B<const X: A>; // error!
 ```
 
-To fix this example, we derive `PartialEq` and `Eq`.
+Only structural-match types (that is, types that derive `PartialEq` and `Eq`)
+may be used as the types of const generic parameters.
```

Comment:
```suggestion
may be used as the type of const generic parameters.
```

Is using `type` here more correct?

Reply:
I \*think\* the current wording is correct. It's actually hard to phrase it haha.